coralproject/talk
# Settings in stream

## What does this PR do?

Moves settings to the stream call

## How do I test this PR?

1. Load the comment stream
2. Check the network tab, there should not be a call to `api/v1/settings`
3. Check the redux store, the settings should be visible in the `config` section of the store
4. All tests should pass
